pkg/meta/base.go

```diff
@@ -573,15 +573,16 @@ func (m *baseMeta) refresh() {
 		m.sesMu.Unlock()
 
 		old := m.getFormat()
-		if format, err := m.Load(false); err != nil {
+		if format, err := m.Load(true); err != nil {
+			logger.Warnf("reload setting: %s", err)
+
+			if errors.Is(err, ErrMetadataVer) {
+				os.Exit(UmountCode)
+			}
+
 			if strings.HasPrefix(err.Error(), "database is not formatted") {
-				logger.Errorf("reload setting: %s", err)
```

Comment:
We do need an outstanding error message to tell users why the mount process exits.